### DISCUSSION TITLE
I'm facing this Traceback problem to run manim in vs code in windows 11

### DISCUSSION THREAD

**Discussion by gazi-faysal-jubayer** [3b1b/manim#1902] [Q&A]
```
Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\tools\Manim\Scripts\manim.exe\__main__.py", line 4, in <module>
  File "C:\tools\Manim\Lib\site-packages\manim\__init__.py", line 15, in <module>
    from ._config import *
  File "C:\tools\Manim\Lib\site-packages\manim\_config\__init__.py", line 22, in <module>
    parser = make_config_parser()
  File "C:\tools\Manim\Lib\site-packages\manim\_config\utils.py", line 115, in make_config_parser
    parser.read(other_files)  # optional files
  File "C:\Python310\lib\configparser.py", line 698, in read
    self._read(fp, filename)
  File "C:\Python310\lib\configparser.py", line 1086, in _read
    raise MissingSectionHeaderError(fpname, lineno, line)
configparser.MissingSectionHeaderError: File contains no section headers.
file: WindowsPath('manim.cfg'), line: 1
'M[CLI]\n'
```

i'm trying to genarate this manim. But it's not working.
I reinstalled manim but it remains same. Even eny kinds of manim isn't running

one of my code:
from manim import*
class FirstExample(Scene) :
    def construct(self):
        
        box = Rectangle(stroke_color = GREEN_C, stroke_opacity=0.7, fill_color=RED_B, fill_opacity=0.5,height=1,width=1)

        self.add(box)
        self.play(box.animate.shift(RIGHT*2), run_time=2)
        self.play(box.animate.shift(UP*3), run_time=2)
        self.play(box.animate.shift(DOWN*5+LEFT*5), run_time=2)
        self.play(box.animate.shift(UP*1.5+LEFT*1), run_time=2)

**Comment by ShifatHasanGNS** [comment]
# If 'manim' and other dependencies are properly installed, then from your issue above I can suggest you to install "Python 3.9" instead of 3.10 or above versions.

### You can follow this video tutorial:
**https://www.youtube.com/watch?v=CYOLQk8GpME&ab_channel=TheoremofBeethoven**

## ** Or, You can follow the instructions below, though I suggest to follow the video. **

### # Install Python 3.9, not the 3.10 or above versions.

### # Install 'Chocolatey':
#### In Power Shell (Admin):
**Run:** `Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))`
 
### # Install ['wget', 'git', 'ffmpeg', 'tinytex'] using 'Chocolatey':
**Run:** `choco install wget git ffmpeg tinytex`

### # Open 'cmd.exe'. Then Run these commands one by one:
`refreshenv`
`python -m pip install manim`
`tlmgr install amsmath babel-english cbfonts-fd cm-super ctex doublestroke dvisvgm everysel`
`tlmgr install fontspec frcursive fundus-calligra gnu-freefont jknapltx latex-bin`
`tlmgr install mathastext microtype ms physics preview ragged2e relsize rsfs`
`tlmgr install setspace standalone tipa wasy wasysym xcolor xetex xkeyval`

### Make a working directory and open it in 'cmd.exe':
`pip freeze`
**Create a Virual-Environment (for example: mce) [mce = Manim Community Edition]:**
`pip install name_of_the_virtual_environment`
`virtualenv mce | virtualenv mgl`
`.\manim_env\Scripts\activate`
### Install ['manim', 'jupyter'] in that Virtual-Environment:
`pip install manim jupyter`

### Download or Copy this example-file `basic.py` into that working directory:
**https://raw.githubusercontent.com/ManimCommunity/manim/main/example_scenes/basic.py**

### Run that `basic.py`:
`manim -pqm basic.py ClassName`

### Run ManimCE:
`manim -pqm filename.py ClassName`

### ManimCE's Official Website:
**https://www.manim.community/**
